src/services/repo/hooks.spec.tsx

```diff
@@ -212,6 +221,7 @@ describe('useUpdateRepo', () => {
           wrapper: wrapper(),
         })
 
+        // @ts-expect-error
```

Comment:
Interested as to what's going on here? is it just that the current hook that returns the mutation hasn't been uploaded to TS yet?